tests/DwitTech.AccountService.Core.Tests/Services/UserServiceTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using DwitTech.AccountService.Core.Dtos;
+using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Services;
+using DwitTech.AccountService.Data.Repository;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Tests.Services
+{
+    public class UserServiceTest
+    {
+        private readonly IConfiguration _configuration;
+        public UserServiceTest()
+        {
+            _configuration = new ConfigurationBuilder().AddInMemoryCollection(new Dictionary<string, string> {
+
+                {"GMAIL_INFO:EMAIL", "johnokpo83@gmail.com"},
+                {"GMAIL_INFO:HOST", "smtp.gmail.com"},
+                {"GMAIL_INFO:PORT", "587"},
+                {"GMAIL_INFO:APP_PASSWORD", "fyvdxxvlsvosecwg"},
+
+            }).Build();
+        }
+
+
+        [Fact]
+        public void CreateUser_ShouldReturnTrue_WhenSuccessful()
```

Comment:
Seems like you're testing a non existent case. Your CreateUser method in UserService has no return value, so how can it return true when successful?

Reply:
Task type of functions return true when it runs through successfully